test/ecto/adapters/clickhouse/structure_test.exs

```diff
@@ -231,4 +231,79 @@ defmodule Ecto.Adapters.ClickHouse.StructureTest do
                """
     end
   end
+
+  describe "structure_load/2" do
+    test "can load structure" do
+      database = "ecto_ch_temp_structure_migrated"
+      opts = [database: database]
+
+      assert :ok = ClickHouse.storage_up(opts)
+      on_exit(fn -> ClickHouse.storage_down(opts) end)
+
+      Application.put_env(:structure_test, Repo,
+        database: database,
+        show_sensitive_data_on_connection_error: true
+      )
+
+      on_exit(fn -> Application.delete_env(:structure_test, Repo) end)
+
+      start_supervised!(Repo)
+
+      assert [1, 2] ==
+               Ecto.Migrator.run(Repo, [{1, Migration1}, {2, Migration2}], :up,
+                 all: true,
+                 log: false
+               )
```

Comment:
We probably don't need to run migrations and dumping schemas in this test. I think we can just load the structure and query the created tables.

Reply:
Done